Documentation/tutorials/saving-and-resuming-tracks.md

```diff
@@ -0,0 +1,58 @@
+# Saving and Resuming a Trial
+
+This tutorial will walk you through the process of saving and resuming a trail. This is useful if you want to save your work and resume it later, or if you want to share your work with someone else.
+This tutorial assumes that you have already completed the [](./loading-and-tracking.md) tutorial or have otherwise loaded and tracked data in AutoscoperM.
+
+## Saving a Trial
+
+### File Structure
+
+In order to automatically load tracks when a trial is loaded in AutoscoperM, the tracks must be saved in a specific file structure. This is the same file structure that is shown in the [](./custom-data.md#automatic-filter-and-tracking-data-loading) section of the [](./custom-data.md) tutorial.
+
+
+The file structure is as follows:
+
+```
+trial name
+├── radiographs
+|   └── ...
+├── volumes
+|   └── ...
+├── calibration
+|   └── ...
+├── xParamters
+|   └── control_settings.vie
+├── Tracking
+|   ├── {trial name}_bone1.tra
+|   ├── {trial name}_bone2.tra
+|   ├── ...
+|   └── {trial name}_bonek.tra
+└── trial name.cfg
+```
+
+### Saving a Trial Configuration
+
+To save a trial configuration, click the `File` menu and select `Save a Trial` or press `Ctrl+Shift+S` on your keyboard. This will open a dialog box that will allow you to select a location to save the trial configuration. The trial configuration will be saved as a `.cfg` file.
+
+
+### Saving Tracks
+
+Inorder for tracks to be automatically loaded when a trial is loaded, tracks must be saved in the default configuration. For more info on the different track saving options, see the [](../user-interface.md#importexport-tracking-options) section of the User Interface page.
```

Comment:
```suggestion
In order for tracks to be automatically loaded when a trial is loaded, tracks must be saved in the default configuration. For more info on the different track saving options, see the [](../user-interface.md#importexport-tracking-options) section of the User Interface page.
```

Reply:
> must be saved in the default configuration

What does this mean exactly ?

Reply:
Where is defined "default configuration" ?

Reply:
This is the default values for saving tracking data in Autoscoper. I updated the line to be: 
```{markdown}
In order for tracks to be automatically loaded when a trial is loaded, tracking data must be saved in the default configuration.

Please see the [](../user-interface.md#importexport-tracking-options) section of the User Interface page for information on the default configuration that Autoscoper uses to save tracking data.
```